ghedesigner/media.py

```diff
@@ -1,58 +1,87 @@
-from dataclasses import InitVar, asdict, dataclass, field
-
-from pygfunction.media import Fluid
+from scp.ethyl_alcohol import EthylAlcohol
+from scp.ethylene_glycol import EthyleneGlycol
+from scp.methyl_alcohol import MethylAlcohol
+from scp.propylene_glycol import PropyleneGlycol
+from scp.water import Water
 
 from ghedesigner.enums import FluidType
 
 
-@dataclass
-class GHEFluid(Fluid):
-    fluid_str: InitVar[str]
-    percent: InitVar[float]
-    temperature: float = 20.0
-    concentration_percent: float = field(init=False)
-    fluid_type: FluidType = field(init=False)
-
-    def __post_init__(self, fluid_str: str, percent: float) -> None:
-        fluid_type_map = {
-            FluidType.ETHYLALCOHOL.name: (FluidType.ETHYLALCOHOL, "MEA"),
-            FluidType.ETHYLENEGLYCOL.name: (FluidType.ETHYLENEGLYCOL, "MEG"),
-            FluidType.METHYLALCOHOL.name: (FluidType.METHYLALCOHOL, "MMA"),
-            FluidType.PROPYLENEGLYCOL.name: (FluidType.PROPYLENEGLYCOL, "MPG"),
-            FluidType.WATER.name: (FluidType.WATER, "WATER"),
-        }
-
+class Fluid:
+    def __init__(self, fluid_name: str, temperature: float = 20, percent: float = 0) -> None:
+        self.name = fluid_name
+        self.fluid_type = self.get_fluid_type(fluid_name)
+        self.temperature = temperature
         self.concentration_percent = percent
```

Comment:
This is just a thin wrapper so we don't have to pass the temperature in to each fluid properties call.